### DISCUSSION TITLE
Catch all Async errors and report to Sentry? (or other error reporting)

### DISCUSSION THREAD

**Discussion by chrisjingram** [socketry/async#288] [Q&A]
Hey,

If I have some tasks such as:

```ruby
Async { PostToSlack.message("hello world") }
```

If these fail, I'm not finding out through my error reporting, Sentry.

How can I catch all errors in Async tasks to report them to Sentry?

For example, in Rails I do something like:

```ruby
  rescue_from StandardError, with: :show_error

  def show_error(exception)
    Sentry.capture_exception(exception)
  end
```

**Comment by ioquatix** [comment]
Does Sentry support log tagging/structured logging?
